core/src/main/java/com/onelogin/saml2/logout/LogoutResponse.java

```diff
@@ -340,12 +345,13 @@ private NodeList query (String query) throws XPathExpressionException {
      * Generates a Logout Response XML string.
      *
      * @param inResponseTo
-     *				InResponseTo attribute value to bet set at the Logout Response. 
+     *				InResponseTo attribute value to bet set at the Logout Response.
      */
-	public void build(String inResponseTo) {
+	public void build(String inResponseTo, String status) {
```

Comment:
Can you leave the public void build(String inResponseTo) method as well?
public void build(String inResponseTo) {
    build(inResponseTo, Constants.STATUS_SUCCESS)
}

And then leave build() as it was
public void build() {
	build(null);
}